core/comparison.cpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+/// @file
+
+#include "scipp/core/comparison.h"
+
+namespace scipp::core {
+
+/**
+ * Tests if the unit, values (and variances where appropriate) of two Variables
+ * are within an absolute tolerance.
+ */
```

Comment:
Follow doxygen style used in rest of the project?

Reply:
Which is?

(I randomly opened around 10 files looking for a Doxygen comment before giving up)

Reply:
C++-style: https://github.com/scipp/scipp/blob/7095c435e72f2a315a6da6c6621855fbfd640010/core/dataset.cpp#L81-L87.